io/disk/arcconf/arcconf_raid_oper.py

```diff
@@ -0,0 +1,252 @@
+            time.sleep(10)
+
+        elif cnt == 4:
+            raid = "10"
+            self.raid_exec(type_name1, disk_data, cnt, raid)
+            time.sleep(10)
+
+            raid = "5"
+            self.raid_exec(type_name1, disk_data, cnt, raid)
+            time.sleep(10)
+
+            raid = "6"
+            self.raid_exec(type_name1, disk_data, cnt, raid)
+            time.sleep(10)
+
+    def raid_exec(self, name1, dsk_data, cnt, raid):
+        """
+        function to create different raid functions
+        """
+        self.log.info(" Creating RAID %s with drives %s"
+                      % (raid, dsk_data))
+        cmd = "echo y | arcconf create %s LOGICALDRIVE %s MAX %s %s" \
+            % (self.crtl_no, name1, raid, dsk_data)
+        self.check_pass(cmd, "Failed to create RAID %s" % raid)
+
+        if raid == 0:
+            self.format_logical(cnt)
+        self.make_fs()
+        time.sleep(10)
+
+        self.log.info(" Deleting RAID %s with drives %s"
+                      % (raid, dsk_data))
+        cmd = "echo y | arcconf delete %s logicaldrive %s" % \
+            (self.crtl_no, self.logicaldrive)
+        self.check_pass(cmd, "Failed to delete RAID %s" % raid)
+
+    def format_logical(self, count1):
+        """
+        format logical drive
+        """
+        self.log.info("Formatting Logical drive %s, having %s drives"
+                      % (self.logicaldrive, count1))
+        cmd = "echo y | arcconf task start %s LOGICALDRIVE %s CLEAR" \
+            % (self.crtl_no, self.logicaldrive)
+        self.check_pass(cmd, "Failed to format Logical Drive")
+        cmd = "arcconf getconfig %s LD" % self.crtl_no
+        self.check_pass(cmd, "Failed to display Logical device details")
+
+    def make_fs(self):
+        """
+        creates filesystem
+        """
+        cmd = "lsscsi  | grep LogicalDrv | awk \'{print $7}\'"
+        drive = self.cmdop_list(cmd, 'N')
+
+        self.log.info("Creating file system on %s" % (drive))
+
+        cmd1 = "echo y | mkfs.ext4 %s" % drive
+        self.check_pass(cmd1, "Failed to create filesystem")
+
+        cmd2 = "mount %s /mnt" % drive
+        self.check_pass(cmd2, "Failed to mount filesystem")
+
+        cmd3 = "cd /mnt && dd if=/dev/random of=Gfile.txt bs=3M count=1"
+        self.check_pass(cmd3, "Failed to create file")
+
+        cmd4 = "cd / && umount %s" % drive
+        self.check_pass(cmd4, "Failed to unmount file")
+
+    def check_pass(self, cmd, errmsg):
+        """
+        Function to check if the cmd is successful or not
+        """
+        if process.system(cmd, timeout=3200, ignore_status=True,
+                          shell=True) != 0:
+            self.fail(errmsg)
+
+    @classmethod
+    def cmdop_list(cls, cmd, disk_type):
+        """
+        Function returns the output of a command
+        """
+        val = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE,
+                               stderr=subprocess.PIPE)
```

Comment:
Spent some quality time experimenting the feasibility of this request. 

here is what i found.  
1. My code uses process.system which doesn't have stdout functionaility. so i will not be able to reuse process.system for code optimisation and i have to use subprocess for stdout functionality.
2. based on python help,  stdout is used in subprocess.  so process library again inherits stdout functionality from subprocess library.  which is more code than me directly calling the subprocess.

with these points i prefer to retain the existing code.